resolver-dns/src/main/java/io/netty/resolver/dns/DefaultDnsCache.java

```diff
@@ -115,8 +115,9 @@ public void clear() {
 
     @Override
     public boolean clear(String hostname) {
-        checkNotNull(hostname, "hostname");
-        return resolveCache.clear(appendDot(hostname));
+        return resolveCache.clear(
+                appendDot(
+                        checkNotNull(hostname, "hostname")));
```

Comment:
nit: maybe merge the lines as the formatting looks a bit strange.

Reply:
revert, use this version:
```
checkNotNull(hostname, "hostname");
return resolveCache.clear(appendDot(hostname));
```